lib/OpenQA/Schema/ResultSet/Comments.pm

```diff
@@ -38,12 +38,7 @@ sub compute_bugref_count {
     my ($self) = @_;
 
     my $comments = $self->search({-not => {job_id => undef}});
-    my %bugrefs;
-    for my $comment ($comments->all) {
-        for my $bug (@{find_bugrefs($comment->text)}) {
-            $bugrefs{$bug} += 1;
-        }
-    }
+    my %bugrefs = map { $_ => 1 } map { @{find_bugrefs($_->text)} } $comments->all;
```

Comment:
It looks like it now doesn't count the bugrefs anymore and only returns whether they exist at all. That might be sufficient but you should rename the function name then.

Reply:
good point. Renamed to "referenced_bugs".